### PR TITLE
Zj1.3/call to action

### PR DESCRIPTION
**Pull request by michaljedro**

### CODE REVIEW

File: app/scss/_call-to-action.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.call-to-action {
+    background-color: #5b605f;
+    height: 20vh;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    & .call-to-action__advert {
+        width: 60vw;
```
Comment (pawel-schmidt) [inline app/scss/_call-to-action.scss:8]: As above. There is a pattern you may follow to put an additional `div` into `.call-to-action` (let's add a `.call-to-action__wrapper` class to it) and the following styles:
```
.call-to-action__wrapper {
  max-width: 1200px;
  margin-left: auto;
  margin-right: auto;
}
```
`max-width` ensures the content is not wider than provided value. When the screen is narrower than provided value, wrapper will shrink.
`margin-x` centers the wrapper.

File: app/scss/_call-to-action.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    & .call-to-action__advert--topic {
+        color: white;
+    }
+    & .call-to-aciton__advert--text {
+        color: white;
+        width: 60vw;
```
Comment (pawel-schmidt) [inline app/scss/_call-to-action.scss:15]: As above ☝️

File: app/scss/_call-to-action.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    & .call-to-action__advert--button {
+        background-color: #f09713;
+        height: 5vh;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: 15vw;
```
Comment (pawel-schmidt) [inline app/scss/_call-to-action.scss:23]: As above ☝️

**Review by pawel-schmidt — CHANGES_REQUESTED** [review]

File: index.html [hunk 4]
```diff
@@ -62,7 +62,16 @@ <h1>Lorem ipsum dolor sit amet.</h1>
...
   </div>
-
+  <div class="call-to-action">
+    <div class="call-to-action__advert">
+      <p class="call-to-action__advert--topic">Lorem ipsum TOPIC sit amet.</p>
+      <p class="call-to-aciton__advert--text">Lorem ipsum TEXT sit amet consectetur adipisicing elit. Saepe placeat
+        quisquam nam sed? Lorem ipsum dolor, sit amet consectetur adipisicing elit. Modi, saepe!</p>
+    </div>
+    <div class="call-to-action__advert--button">
+      <a href="" class="call-to-action__advert--button-text">Lorem Ipsum</a>
```
[the remark this answers is not in the document]
Comment (michaljedro) [reply index.html]: ok. i missed up this

File: app/scss/_call-to-action.scss [hunk 5]
```diff
@@ -0,0 +1,34 @@
+.call-to-action {
+    background-color: #5b605f;
+    height: 20vh;
```
[the remark this answers is not in the document]
Comment (michaljedro) [reply app/scss/_call-to-action.scss]: ok. i changed it

File: app/scss/_call-to-action.scss [hunk 6]
```diff
@@ -0,0 +1,32 @@
+.call-to-action {
+    background-color: #5b605f;
+    color: white;
+    & .call-to-action__wrapper {
```
Comment (pawel-schmidt) [inline app/scss/_call-to-action.scss:4]: Ampersand (`&`) is redundant here. In short, these both are working similarly:
```
.a {
  .b {
    color: white;
  }
}
```

```
.a {
  & .b {
    color: white;
  }
}
```
and will produce `.a .b { color: white; }`. Maybe you wanted to do it in this way:
```suggestion
&__wrapper {
```
?

File: app/scss/_call-to-action.scss [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+        display: flex;
+        justify-content: space-evenly;
+        padding: 100px;
+    }
+    & .call-to-action__advert {
```
Comment (pawel-schmidt) [inline app/scss/_call-to-action.scss:9]: As above ☝️  and as below 👇

**Review by pawel-schmidt — CHANGES_REQUESTED** [review]

File: app/scss/_call-to-action.scss (hunk 1 above)
Comment (michaljedro) [reply app/scss/_call-to-action.scss]: ok. i added it to my code

File: app/scss/_call-to-action.scss (hunk 2 above)
Comment (michaljedro) [reply app/scss/_call-to-action.scss]: i dont have got it in my code

File: app/scss/_call-to-action.scss (hunk 3 above)
Comment (michaljedro) [reply app/scss/_call-to-action.scss]: ok. i centered that with margin

File: app/scss/_call-to-action.scss (hunk 6 above)
Comment (michaljedro) [reply app/scss/_call-to-action.scss:4]: ok. i changed that

File: app/scss/_call-to-action.scss (hunk 7 above)
Comment (michaljedro) [reply app/scss/_call-to-action.scss:9]: changed

**Review by pawel-schmidt — APPROVED** [review]

**Pull request merged**